statistics/read-rtdb.js

```diff
@@ -1,15 +1,88 @@
-const serviceAccount = require("../../universalgamemaker-firebase-adminsdk.json");
+//const serviceAccount = require("../../universalgamemaker-firebase-adminsdk.json");
```

Comment:
Reset this file or remove changes here.

This shouldn't be visible in this PR.